src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -201,6 +201,9 @@ object StringUtils {
         return "$allButLast$delimiterColor, and ${this[lastIndex]}"
     }
 
+    fun String.pluralize(number: Int, plural: String? = null, withNumber: Boolean = false) =
```

Comment:
can we remove plural from the extension function? the format cactus.plural(3, cactii) feels odd. for "non s plurals" we have two strings, and having one as extension function parameter and one as normal parameter feels odd. for that case we should still just use the non extension function